webtau-groovy/src/main/groovy/com/twosigma/webtau/cfg/WebTauGroovyFileConfigHandler.groovy

```diff
@@ -44,25 +52,40 @@ class WebTauGroovyFileConfigHandler implements WebTauConfigHandler {
 
         validateEnv(cfg, workingDir, configPath)
 
-        def groovy = GroovyRunner.createWithoutDelegating(cfg.workingDir)
-
-        ConfigSlurper configSlurper = new ConfigSlurper(cfg.env)
-        def configScript = groovy.createScript(configPath.toUri().toString(), new ConfigBinding())
-
-        def parsedConfig = configSlurper.parse(configScript)
-        cfg.acceptConfigValues("config file", convertConfigToMap(parsedConfig))
+        ConfigObject parsedConfig = parseConfig(cfg, configPath)
 
         if (!parsedConfig) {
             return
         }
 
+        cfg.acceptConfigValues("config file", convertConfigToMap(parsedConfig))
+
         setupHttpHeaderProvider(parsedConfig)
         setupBrowserPageNavigationHandler(parsedConfig)
         setupReportGenerator(parsedConfig)
         setupPageElementGetSetValueHandlers(parsedConfig)
         setupTestListeners(parsedConfig)
     }
 
+    private static ConfigObject parseConfig(WebTauConfig cfg, Path configPath) {
+        try {
+            def groovy = GroovyRunner.createWithoutDelegating(cfg.workingDir)
+
+            ConfigSlurper configSlurper = new ConfigSlurper(cfg.env)
+            def configScript = groovy.createScript(configPath.toUri().toString(), new ConfigBinding())
+
+            def parsedConfig = configSlurper.parse(configScript)
+            return parsedConfig
+        } catch (Throwable e) {
```

Comment:
this seems overly generic -- is there an `Error` being thrown that you think we can recover from?

Reply:
This is basically to avoid repl crash when config is bad. It could be any number of Groovy parsing exceptions.

Reply:
Exceptions yes, but Errors?

Reply:
got it - changed.

Reply:
it's still `Throwable`, should be `Exception` , no?

Reply:
I need coffee